apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -810,6 +813,7 @@ const CTA = ({ data }: { data: GetByViewerResponse }) => {
 
   const profileOptions = data.profiles
```

Comment:
We need to disable the `CreateButton` if we don't have any profileOptions. Now I am getting an error when clicking on the 'New' button `Uncaught TypeError: Cannot read properties of undefined (reading 'slug')`

Reply:
Never even thought of this - nice spot